setup.py

```diff
@@ -2,11 +2,15 @@
 
 readme = open('README.md').read()
 
+with open('requirements.txt') as requirements_file:
+    requirements = requirements_file.read().splitlines(False)
+
 setup(
```

Comment:
Yeah! I picked it up from https://github.com/coala-analyzer/coala/blob/master/setup.py